app/features/tasks.py

```diff
@@ -1 +1,66 @@
+import pickle 
+import os.path
+from googleapiclient.discovery import build
+from google_auth_oauthlib.flow import InstalledAppFlow
+from google.auth.transport.requests import Request
+
+class tasks:
+
+    def __init__(self):
+        self.SCOPES = ['https://www.googleapis.com/auth/tasks']
+        self.creds = None
+        if os.path.exists('token_calendar_v3.pickle'):
+            with open('token_calendar_v3.pickle', 'rb') as token:
+                self.creds = pickle.load(token)
+            
+        if not self.creds or not self.creds.valid:
+            if self.creds and self.creds.expired and self.creds.refresh_token:
+                self.creds.refresh(Request())
+            else:
+                flow = InstalledAppFlow.from_client_secrets_file('credentials.json', 'https://www.googleapis.com/auth/tasks')
+                self.creds = flow.run_local_server(port=0)
+            
+            with open('token.pickle', 'wb') as token:
+                pickle.dump(self.creds, token)
+            
+        self.service = build('tasks', 'v1', credentials=self.creds)
+
+    def list_tasks(self):
+        results = self.service.tasklists().list().execute()
+        items = results.get('items', [])
+
+        if not items:
```

Comment:
Instead of printing the list of tasks, can you store them in an array and return that instead. That way, we can handle the array however we want. The current function will print the tasks into console and return nothing, but let's say we want to take all our tasks and sort them by alphabetical order or based off when it was created we have that option